### DISCUSSION TITLE
The converted coordinates appear to be abnormal and cannot be used in three.js

### DISCUSSION THREAD

**Discussion by nightISend** [d3/d3#3839] [Q&A]
const projection =d3
        .geoMercator()//设置投影方法，球形墨卡托投影
        .scale(84)//缩放
        .center([108.5525,34.3227])//中心
        .translate([0,0])//移动

This is a set of coordinates
[103.953779, 28.600906] before convert
[-6.74207711777089, 9.841509178425873]after converted

and three.js report
THREE.WebGLAttributes: Unsupported buffer data format: -6.74207711777089, 9.841509178425873,

why?

**Comment by curran** [comment]
It's difficult to help without seeing the full code. Can you please share a running example so we can help debug?

For example, it's unclear what you mean by "convert". Do you mean passing it through the projection? Also, it's unclear how that is passed into ThreeJS.

**Reply by nightISend** [reply]
I don't know why I can't put pictures, so I could only explain my code in wards.

I create a projection and put a coordinate([100,30]) in the projection, then return result which is different with Tutorials I learned, is that right?

const projection =d3
        .geoMercator()
        .scale(84)
        .center([108.5525,34.3227])
        .translate([0,0])
     console.log(projection([100,30]))//result=[-12.538619879252451, 7.489320921373903]
